### PR TITLE
allow set self signed certs

### PR DESCRIPTION
**Pull request by jlvcm**

### CODE REVIEW

File: actual/api/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import datetime
+from enum import verify
```
Comment (bvanelli) [inline actual/api/__init__.py:4]: Was this imported by mistake?

Comment (jlvcm) [reply actual/api/__init__.py]: yup, you'r right, sorry about that, fixed :pray:

**Comment by bvanelli** [comment]
Tests are also failing, try:

```diff
diff --git a/tests/test_api.py b/tests/test_api.py
index e16805e..402b28d 100644
--- a/tests/test_api.py
+++ b/tests/test_api.py
@@ -33,6 +33,7 @@ def test_rename_delete_budget_without_file():
 @patch("requests.post", return_value=RequestsMock({"status": "error", "reason": "proxy-not-trusted"}))
 def test_api_login_unknown_error(_post):
     actual = Actual.__new__(Actual)
+    actual.cert = True
     actual.api_url = "localhost"
     with pytest.raises(AuthorizationError, match="Something went wrong on login"):
         actual.login("foo")
```

**Comment by bvanelli** [comment]
Merged, will come on the next release.

Thanks for your contribution.

**Pull request merged**